docs/products/compute/kubernetes/guides/upgrade-kubernetes-version/index.md

```diff
@@ -13,35 +13,27 @@ external_resources:
 
-## Check the Kubernetes Version on an LKE Cluster
-
-Navigate to the **Kubernetes** page in the [Cloud Manager](http://cloud.linode.com) to see a list of all LKE clusters on your account (see [Manage Kubernetes Clusters](/docs/products/compute/kubernetes/guides/manage-clusters/)).
-
-![Check Kubernetes Version](lke-version.png)
-
-The current Kubernetes version of each LKE cluster is listed within the *Version* column. If an upgrade is available, an **Upgrade** button is displayed next to the version number.
-
-Alternatively, if you would like to get the current local version of Kubernetes on your cluster through kubectl, use the following command:
+{{< note >}}
+When upgrading a minor version of Kubernetes on LKE, you must upgrade to the next consecutive version. If the cluster is multiple versions behind, each version upgrade must be completed individually. For instance, if the current version on the cluster is 1.23.x and the latest available version on LKE is 1.25.x, the cluster must be upgraded to v1.24.x *before* upgrading to 1.25.x.
+{{< /note >}}
 
-```command
-kubectl version | grep Server
-```
+## Upgrade Process
 
-## What Does an Upgrade Look Like
+When Kubernetes is upgraded on an LKE cluster (including both patch versions and minor versions), the control plane components are automatically upgraded. New worker nodes are also created using the new version. In most cases, there is no downtime or impact to performance for this part of the upgrade.
 
-When upgrading a Kubernetes cluster on LKE, it is important to keep in mind that all nodes within the cluster will need to be [recycled](/docs/products/compute/kubernetes/guides/upgrade-kubernetes-version/
-) on a rolling basis. In other words, the nodes within your cluster are taken down and upgraded one at a time, to help ensure that the process completes without downtime.
+Existing worker nodes are only automatically upgraded when you use the Cloud Manager and select **Recycle All Nodes** during the upgrade process. This recycles all nodes on a rolling basis so that only a single node is down at any time. Since this means there is one less worker node during the upgrade process, it can affect performance and might not be preferred for production applications. It is strongly recommended that steps are taken to ensure that there is enough space on all nodes to accommodate for this temporary shift in resources. If a cluster of three nodes cannot briefly support the resources demands of an application using only two nodes, then the upgrade process may result in unintended application downtime. To overcome this, you can temporarily add additional worker nodes and perform either an *in-place* or *out-of-place* upgrade to avoid any performance impact. For more details on these approaches, see [Upgrade Worker Nodes](#upgrade-worker-nodes).
```

Comment:
Should we mention that this option is not just limited to Cloud Manager?

Reply:
You're right, I think we should clarify that here. I reworked this section a bit to mention all three upgrade options and then link out to the upgrade worker nodes section for more details. Since it is important to mention that worker nodes are automatically upgraded in the Cloud Manager in most cases (if the user doesn't click Cancel), I moved that information to a warning note.